containers/CompoundAndEarn/ListItem/index.js

```diff
@@ -159,6 +161,35 @@ const ListItem = ({
 
   const userBoost = `${(boost ? boost * 1.0 : 1.0).toFixed(2)}x`;
   const highlighted= action?.actionType === 'Details' && (AVAXBalance !== 0 || (userData?.userDepositedLP || 0) > 0)
+
+  const renderCaution = (harvestInfo) => {
+    if (harvestInfo.error) {
+      return (
+        <Grid item xs={12}>
+          <Caution>
+            <Typography variant="caption">
+              This pool is not currently harvesting. We are aware of the issue
+              and are working on it.
+            </Typography>
+          </Caution>
+        </Grid>
+      );
+    } else if (harvestInfo.fulfillThreshold) {
```

Comment:
the logic is inverted here, the warning should show when fulfillthreshold is false